script.js

```diff
@@ -17,15 +17,17 @@ const fr = 10
 // that may take a little while to finish
 function preload() {
 	modes = [
+    new GameMode('☰', '', new Options()),
 		new GameMode('Grade One', '', new Game(gradeOnePhrases, '🍪', 5, loadImage('assets/candy.jpg'))),
-		new GameMode('Grade Ten+', '', new Game(standardPhrases, '_', 3, loadImage('assets/candy.jpg'))),
+		new GameMode('Grade Ten+', '', new Game(standardPhrases, '_', 3, loadImage('assets/candy.jpg')))
 	]
 	preLoadSoundFiles()
 }
 
 function setup() {
 	// Make the drawing canvas as big as the window
 	createCanvas(windowWidth, windowHeight)
+  document.getElementById("file-selector").addEventListener('change', (event) => {addNewCsvPhraseList(event)})
```

Comment:
"file-selector" should be a constant somewhere since it is referenced more than once.

Reply:
`options.js` line 8